src/Uno.UI/UI/Xaml/Media/PointCollection.cs

```diff
@@ -9,30 +9,102 @@ namespace Windows.UI.Xaml.Media
 {
 	public partial class PointCollection : IEnumerable<Point>, IList<Point>
 	{
-		private List<Point> _coords;
+		private readonly List<Point> _points;
+		private readonly List<Action> _changedCallbacks = new List<Action>();
 
 		public PointCollection()
 		{
-			_coords = new List<Point>();
+			_points = new List<Point>();
 		}
 
 		public PointCollection(IEnumerable<Point> coordinates)
 		{
-			_coords = coordinates.ToList();
+			_points = coordinates.ToList();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'coordinates' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5633443)